tests/blackbox/setup/setup.ts

```diff
@@ -1,17 +1,19 @@
 /* eslint-disable no-console */
 
+import axios from 'axios';
+import { spawn, spawnSync } from 'child_process';
+import { writeFileSync } from 'fs';
 import knex from 'knex';
 import { Listr } from 'listr2';
-import vendors from '../common/get-dbs-to-test';
+import { clone } from 'lodash';
+import path from 'path';
+import * as common from '../common';
 import config, { getUrl } from '../common/config';
-import global from './global';
-import { spawn, spawnSync } from 'child_process';
-import axios from 'axios';
-import { writeFileSync } from 'fs';
+import vendors from '../common/get-dbs-to-test';
 import { awaitDatabaseConnection, awaitDirectusConnection } from '../utils/await-connection';
-import * as common from '../common';
-import { clone } from 'lodash';
+import global from './global';
 
+const apiPath = path.join(__dirname, '../../../dist/cli');
```

Comment:
Maybe this can be combined with https://github.com/directus/directus/pull/18122/files#diff-a82b3abb249145626e26abf82a25f76f0f143492484aaa436a92a7f2ff9981e3R75 as an exported variable from somewhere, but it might also be overkill, so just throwing this thought out there. 😄

Reply:
We were thinking the same 😸👍